### ISSUE TITLE
How to configure multi-page entry

### ISSUE THREAD

**Issue by sbfkcel** [fastify/fastify-vite#113]
### Prerequisites

- [X] I have written a descriptive issue title
- [X] I have searched existing issues to ensure the feature has not already been requested

### 🚀 Feature Proposal

I tried to configure multi-page entry which is currently not supported

Or did I not find the corresponding documentation?

### Motivation

Many times, multi-page entry is very important. Vite itself also allows the configuration of multi-page entrances.

**Comment by onlywei** [comment]
How do you configure multiple entry points with vite?

**Comment by wimbarelds** [comment]
> How do you configure multiple entry points with vite?

https://vitejs.dev/guide/build#multi-page-app

**Comment by Dan-Shields** [comment]
This issue is unfortunately preventing us from using the plugin. It feels like its scope is stretched by offering SSR support which then that means it doesn't support all configuration options in the vite config (rollup inputs are manually specified for MPAs).

I'm going to work on an MPA plugin for vite and report back if I get anywhere.

**Comment by galvez** [comment]
@Dan-Shields @onlywei indeed, that should be a somewhat straightforward change to implement, where we no longer consider `client/index.js` but rather a list as defined by the Vite configuration.

**Comment by galvez** [comment]
I still have a massive PR from @onlywei to review over the weekend, but let's see 😄 

@onlywei if you want to be onboarded I'll be free Saturday afternoon, let me know!

**Comment by onlywei** [comment]
@galvez I'm unfortunately not free on Saturday afternoon. When is your next availability after that?

**Comment by galvez** [comment]
Added to roadmap.

**Comment by ChaosExAnima** [comment]
hi! i'm trying to get this working for my app but unable to get it to work. i checked and i'm unable to find mention of this in the roadmap.

is multiple index.html entrypoints supported yet @onlywei? thanks!

**Comment by onlywei** [comment]
> hi! i'm trying to get this working for my app but unable to get it to work. i checked and i'm unable to find mention of this in the roadmap.
> 
> is multiple index.html entrypoints supported yet [@onlywei](https://github.com/onlywei)? thanks!

Thanks for the reminder. Unfortunately we have not added this feature yet.

I can't say for certain when it will be added. I'm currently working through fixing a lot of CI automation so it's easier to actually work in this repo (and so others can help contribute as well). Once that's done we will re-visit the roadmap for feature enhancements.

**Comment by ChaosExAnima** [comment]
understood. can you reopen this issue until this is added to the roadmap?
